### DISCUSSION TITLE
Is there any way to limit the screensize of the "RemoteApp" ?

### DISCUSSION THREAD

**Discussion by bmullan** [kimmknight/remoteapptool#120] [General]
When I launch a remoteapp I've configured on Win11 using RemoteAppTool the remoteapp starts ok but in full screen
how can  I limit the remoteapps screen size?

thanks

**Comment by kimmknight** [comment]
G'day Brian, haven't heard from you for a while 🙂

I don't believe there are any RDP settings on the host or client that can do this. Do you want it so that when a RemoteApp is maximized it doesn't take up the full screen on the client?

**Reply by bmullan** [reply]
@kimmknight 

Yes, remoteapps start in full screen mode & reducing its size can be tough.  Window control
to reduce can lead to a > max size where the control minimize, maximize, terminate become inoperable

  On Ubuntu I have to use the keyboard windows key which presents all active windows which then lets you terminate the remoteapp

Using xfreerdp3 & /h:640 /w:480 helps alot but its not 100%

Also, I've found that logging into the Windows 11 Desktop & running thet target app. Once & resizing it there, then terminating it, Windows seems to do subsequent launches in that reduced size & thus subsequent remoteapp executions stay at the reduced size.

I just think having to do those extra steps 'should' be do-able somehow w either xfreerdp or remoteapptool.

That's what led me to try /h: /w: cmd options

**Reply by kimmknight** [reply]
> Also, I've found that logging into the Windows 11 Desktop & running thet target app. Once & resizing it there, then terminating it, Windows seems to do subsequent launches in that reduced size & thus subsequent remoteapp executions stay at the reduced size.

That sounds annoying. If your workaround is too cumbersome (eg, if you have to do it for large number of users) I'd suggest using a tool like RegShot on a host (a test VM) to capture the setting that is being changed when the app first runs. If it's a reg key that is being changed, there a number of options (such as group policy) to deploy the key in the user profile on the host for each new user that logs in.
